src/deflate.js

```diff
@@ -1,5 +1,7 @@
 // @flow
 
+import isPlainObject from './isPlainObject';
```

Comment:
I would rather use `is-plain-object` here

https://www.npmjs.com/package/is-plain-object

Reply:
Really? Your package now has zero deps! It's a rare feat and something I think should be preserved when possible. :)

Reply:
What is the value in having no dependencies vs duplicating code logic across multiple packages?

Reply:
🤷‍♂️

I've gotten a bit more reserved on slapping packages in my project when I don't need to. LeftPadGate, having control and that.

Reply:
But I'll change to `is-plain-object`, sec.